tests/unit/python/foglamp/common/test_common_utils.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Unit tests for common utils """
+
+import pytest
+from foglamp.common import utils as common_utils
+from collections import Counter
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("common", "utils")
+class TestCommonUtils:
+    @pytest.mark.parametrize("test_string, expected", [
+        ("Gabbar&Gang", False),
+        ("with;Sambha", False),
+        ("andothers,halkats", False),
+        ("@Rampur", False),
+        ("triedloot/arson", False),
+        ("For$Gold", False),
+        ("Andlot{more", False),
+        ("Andmore}", False),
+        ("Veeru+Jai", False),
+        ("Gaonwale,Thakur", False),
+        ("=resisted", False),
+        ("successfully:", False),
+        ("any attack!", True),
+    ])
+    def test_check_reserved(self, test_string, expected):
+        actual = common_utils.check_reserved(test_string)
+        assert expected == actual
+
+    @pytest.mark.parametrize("test_input_old, test_input_new, expected", [
+        ({'a': 1, 'b': 2, 'c': 3}, {'a': 11, 'b': 22, 'c': 33}, ['a', 'b', 'c']),
+        ({'a': 1, 'b': 2, 'c': 3}, {'a': 11, 'b': 22, 'd': 44}, ['a', 'b', 'd']),
+        ({'a': 1, 'b': 2, 'c': 3}, {'a': 11, 'b': 22}, ['a', 'b']),
+        ({'a': 1, 'b': 2, 'c': 3}, {'d': 11, 'e': 22, 'c': 33}, ['d', 'e', 'c'])
+    ])
+    def test_get_diff(self, test_input_old, test_input_new, expected):
```

Comment:
We  are not using this until now from actual code, which is introduced in this PR only. So let's remove that and this test as well.

Reply:
done